core/planner.c

```diff
@@ -354,6 +355,9 @@ static void create_dive_from_plan(struct diveplan *diveplan, struct dive *dive,
 		finish_sample(dc);
 		dp = dp->next;
 	}
+
+	dive->dc.last_manual_time.seconds = last_manual_point;
+
```

Comment:
could be cleaner to remove either the empty line before or after the `dive->dc...` line.

Reply:
done